### ISSUE TITLE
How to draw a rounded rectangular border for the outside box?

### ISSUE THREAD

**Issue by miraquddus** [dusadrian/venn#16]

**Comment by dusadrian** [comment]
You need an external package called `grid`, for instance:
```r
library(venn)
library(grid)

venn(5, bpx = FALSE)
grid.roundrect(width = 0.98, height = 0.98)
```

**Comment by miraquddus** [comment]
Hi Adrian,

Thanks for your prompt guidance. When I applied the grid command, it draws a rounded rectangle on a separate frame. I am not sure how to incorporate in my existing code. 
```r
library('venn')
library('ggplot2')
library('grid')
grid.roundrect(width = 0.98, height = 0.98)
intersections <- c('~A~B~C', '~A~BC', '~AB~C', '~ABC', 'A~B~C', 'A~BC', 'AB~C', 'ABC')
venn(intersections, zcolor = c("black","white","white","white", "white","grey100","white","blue"), sncs = .85, ilabels = TRUE, box = FALSE)
```

**Comment by dusadrian** [comment]
The `grid.roundrect()` command should be typed last. First draw the Venn diagram, and only after draw the rounded corners rectangle:
```r
intersections <- c('~A~B~C', '~A~BC', '~AB~C', '~ABC', 'A~B~C', 'A~BC', 'AB~C', 'ABC')
venn(intersections, zcolor = c("black","white","white","white", "white","grey100","white","blue"), sncs = .85, ilabels = TRUE, box = FALSE)
grid.roundrect(width = 0.98, height = 0.98)
```

**Comment by miraquddus** [comment]
The rectangle is sitting in front of the venn diagram. In order to make it meaningful, two options:
a) make the grid diagram transparent
b) push the grid window to the back of the 
````{r}
library('venn')
library('grid')
venn(3, bpx = FALSE, box = FALSE)
grid.roundrect(width = 0.6, height = 0.6)
````
![image_2022-09-06_180442868](https://user-images.githubusercontent.com/59379612/188748648-0e153a83-7659-4b25-95eb-1b050d110f2b.png)

**Comment by dusadrian** [comment]
@miraquddus, this is not happening on my computer, neither on MacOS nor in Windows.
And, nevertheless, I have no control over the functions in package `grid`, you need to read their manual.
I see that you have repeatedly used the argument `bpx = FALSE`, which is not part of the function `venn()` formal arguments (and, AFAIK, nor is it a formal option for the base `par`).
Is there a special reason you are using that particular thing?

![windows](https://user-images.githubusercontent.com/915070/188802440-ebcaddaa-2200-457d-b088-a0a140ae1ec3.png)
![macos](https://user-images.githubusercontent.com/915070/188802446-cdf6bf4f-2f11-45e7-93a3-5991e1737ed9.png)

**Comment by miraquddus** [comment]
Thanks for your prompt reply and help to solve the issue. It works now. I really appreciate it.
